packages/flutter/test_fixes/material.dart.expect

```diff
@@ -898,4 +898,11 @@ void main() {
   themeData = ThemeData(bottomAppBarTheme: BottomAppBarTheme(color: Colors.green));
   themeData = ThemeData.raw(bottomAppBarTheme: BottomAppBarTheme(color: Colors.green));
   themeData = ThemeData.copyWith(bottomAppBarTheme: BottomAppBarTheme(color: Colors.green));
+
+  // Changes made in https://github.com/flutter/flutter/pull/111706
+  Scrollbar scrollbar = Scrollbar(trackVisibility: true);
+  bool nowShowing = scrollbar.trackVisibility;
+  ScrollbarThemeData scrollbarTheme = ScrollbarThemeData(trackVisibility: nowShowing);
```

Comment:
`trackVisibility` in `ScrollbarThemeData` is type of `MaterialStateProperty<bool?>?`, can this fix works well?

Reply:
Yeah, it's not a 100% fix, but we did the same transformation for `isAlwaysShown` ➡️  `thumbVisibility`.

In general, if we can point folks in the right direction, I think it's helpful. :)